packages/mds-policy/server.ts

```diff
@@ -16,7 +16,7 @@ import { ApiServer } from '@mds-core/mds-api-server'
 import { api } from './api'
 
 const {
-  env: { npm_package_name, PORT = 4007 }
+  env: { npm_package_name, PORT = 4003 }
```

Comment:
Interesting that these ports are being swapped.

Reply:
All the APIs have been setup to use distinct ports so you can run them locally concurrently. `mds-policy` has historically used port `4003`. `mds-policy-author` is assigned `4007`.